static/app/views/discover/table/tableView.tsx

```diff
@@ -189,13 +189,23 @@ function TableView(props: TableViewProps) {
       }
 
       let target;
-      if (dataRow['event.type'] === 'transaction') {
-        if (dataRow.trace === null) {
+
+      if (dataRow.trace === null) {
+        if (dataRow['event.type'] === 'transaction') {
           throw new Error(
             'Transaction event should always have a trace associated with it.'
           );
         }
 
+        const project = dataRow.project || dataRow['project.name'];
+
+        target = {
+          // NOTE: This uses a legacy redirect for project event to the issue group event link
+          // This only works with dev-server or production.
```

Comment:
Just tested this out a bit on my `dev-ui` and it seems to work. Is this comment still necessary? I didn't get any weird infinite redirects or anything.

Reply:
Also should we wrap the `to={target}` in the styled link with `normalizeUrl()`?

Reply:
Good point, I wrapped the path from the `trace:null` case with `normalizeUrl()` the other path already uses it.